### ISSUE TITLE
[问题/Issue] 章节2.2.5：章节内容中代码与代码不一致，易造成混淆

### ISSUE THREAD

**Issue by 1579088993** [datawhalechina/happy-llm#101]
### 1. 遇到问题的章节 / Affected Chapter

2.2.5 Encode中

### 2. 具体问题描述 / Problem Description

章节内容中代码

`class EncoderLayer(nn.Module):
  '''Encoder层'''
    def __init__(self, args):
        super().__init__()
        # 一个 Layer 中有两个 LayerNorm，分别在 Attention 之前和 MLP 之前
        self.attention_norm = LayerNorm(args.n_embd)
        # Encoder 不需要掩码，传入 is_causal=False
        self.attention = MultiHeadAttention(args, is_causal=False)
        self.fnn_norm = LayerNorm(args.n_embd)
        self.feed_forward = MLP(args)
`
建议把MLP的参数args修改为和code中一样的args.dim, args.dim, args.dropout

### 3. 问题重现材料 / Reproduction Materials

<img width="1694" height="505" alt="Image" src="https://github.com/user-attachments/assets/3831b038-e157-4eef-8298-eed75a4e12f3" />

### 确认事项 / Verification

- [x] 此问题未在过往Issue中被报告过 / This issue hasn't been reported before

**Comment by logan-zou** [comment]
已修复，感谢提醒
